test/testscripts/issue120.jl

```diff
@@ -1,5 +1,7 @@
-using Gadfly, DataFrames, Compat, CSV
+using Gadfly, DataFrames, Compat, DelimitedFiles
 
 set_default_plot_size(6inch, 3inch)
 
-plot(CSV.read(joinpath(dirname(@__DIR__), "data","issue120.csv")), x=:x1, y=:x2)
+dlm = readdlm(joinpath(dirname(@__DIR__), "data","issue120.csv"), ',')
```

Comment:
or in this style (using `header=true`)
```julia
dlm, varnames =  readdlm( joinpath(...), ',', header=true)
```
which will simplify the next line (no convert needed for `dlm`)

Reply:
Ignore my deleted remark on #120, I realise now #120 was a test about precision.